Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
+RUN pip install poetry
+WORKDIR /turbot
+COPY . ./
+RUN poetry config virtualenvs.create false \
```

Comment:
I'm surprised this works. Doesn't this tell poetry not to create a venv if one doesn't exist? So does one exist already somehow I guess?

Reply:
it just uses the "system" python

Reply:
Fancy.